dataprofiler/data_readers/csv_data.py

```diff
@@ -71,15 +75,15 @@ def __init__(self, input_file_path=None, data=None, options=None):
         #  _selected_columns: columns being selected from the entire dataset
         #  _header: any information pertaining to the file header.
         self._data_formats["records"] = self._get_data_as_records
-        self.SAMPLES_PER_LINE_DEFAULT = options.get("record_samples_per_line", 1)
-        self._selected_data_format = options.get("data_format", "dataframe")
-        self._delimiter = options.get("delimiter", None)
-        self._quotechar = options.get("quotechar", None)
-        self._selected_columns = options.get("selected_columns", list())
-        self._header = options.get("header", "auto")
-        self._checked_header = "header" in options and self._header != "auto"
-        self._default_delimiter = ","
-        self._default_quotechar = '"'
+        self.SAMPLES_PER_LINE_DEFAULT: int = options.get("record_samples_per_line", 1)
+        self._selected_data_format: str = options.get("data_format", "dataframe")
+        self._delimiter: Optional[str] = options.get("delimiter", None) 
```

Comment:
we should remove the extra spaces at the end of these too, is pre-commit setup or did it not run?

Reply:
Looks like pre-commit didn't run for some reason. I manually ran it and pushed formatting changes.